src/java.base/share/classes/sun/security/provider/certpath/RevocationChecker.java

```diff
@@ -740,9 +746,37 @@ private void checkOCSP(X509Certificate cert,
                         null, -1);
                 }
 
+                List<Extension> tmpExtensions = new ArrayList<Extension>();
+                if (rp.ocspNonce) {
+                    if (nonce == null) {
+                        try {
+                            // create the 16-byte nonce by default
+                            Extension nonceExt = new OCSPNonceExtension(DEFAULT_NONCE_BYTES);
+                            tmpExtensions.add(nonceExt);
+
+                            if (debug != null) {
+                                debug.println("Default nonce has been created in the OCSP extensions");
+                            }
+                        } catch (IOException e) {
+                            throw new CertPathValidatorException("Failed to create the default nonce " +
+                                    "in OCSP entensions");
```

Comment:
Typo: s/entensions/extensions/

Also, use the `CertPathValidatorException(String, Throwable)` ctor instead and pass the `IOException` as the 2nd parameter.

Reply:
Fixed.